README.md

```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
 import alpaca_trade_api as tradeapi
 
 api = tradeapi.REST()
+# all of these examples work
 aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from='2019-01-01', to='2019-02-01').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01'), to='2019-01-10').df
```

Comment:
we should keep it consistent to use to='2019-02-01' with existing example.

Reply:
fixed